components/Cards/index.js

```diff
@@ -17,3 +17,55 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+    axios.get(`https://lambda-times-backend.herokuapp.com/articles`)
+         .then(res => {
+             console.log(res)
+             res.data.articles['javascript'].forEach(article => {
```

Comment:
We have previously seen many examples of arrays of objects (AOO).  AOO's are useful as you can loop over them.  In this project though we have an Object of arrays.  It's a bit different.  Is there a way you can programmatically loop through everything in an object?